app/models/database_backup.rb

```diff
@@ -99,4 +94,16 @@ def schedule_name
   def backup_file_name
     File.join(region_name, schedule_name, "#{region_name}_#{Time.now.utc.strftime("%Y%m%d_%H%M%S")}.backup")
   end
+
+  private
+
+  def current_db_opts
+    current = Rails.configuration.database_configuration[Rails.env]
+    {
+      :hostname => current["host"],
+      :dbname   => current["database"],
+      :username => current["username"],
+      :password => current["password"]
```

Comment:
No, `Rails.configuration.database_configuration[Rails.env]` decrypts it automatically via https://github.com/Fryguy/manageiq/blob/remove_miq_db_config/config/environments/patches/database_configuration.rb#L13